eldritch/ancient_ones.py

```diff
@@ -0,0 +1,169 @@
+import abc
+
+from eldritch import events, places, characters, monsters, values
+
+
+class AncientOne(metaclass=abc.ABCMeta):
+  # pylint: disable=unused-argument
+  def __init__(self):
```

Comment:
Looks like all ancient ones need a name, a max_doom, and an attributes. Let's make them part of the `__init__` method to makes sure they always get set.

Reply:
Done